stackslib/src/chainstate/stacks/mod.rs

```diff
@@ -731,10 +731,6 @@ pub struct TenureChangePayload {
     pub cause: TenureChangeCause,
     /// The ECDSA public key hash of the current tenure
     pub pubkey_hash: Hash160,
-    /// The Stacker signature
-    pub signature: ThresholdSignature,
-    /// A bitmap of which Stackers signed
-    pub signers: Vec<u8>,
```

Comment:
Do we want this bitmap for blocks?

Reply:
Yes, but we can add that back in via a separate PR.